### PR TITLE
Have compile-models add quiz data to enclave

### PR DESCRIPTION
**Pull request by mjoss17**

### CODE REVIEW

**Comment by rnathuji** [comment]
I didn't notice this in earlier PRs, but code coverage is not being calculated correctly by CI (even in local environments with a fresh `venv`) which is why the diff here is marked up so much by codecov. I'll investigate as part of code review and push up a fix once I figure out what the problem is.

File: enclave_mgmt/compile_models.py [hunk 1]
```diff
@@ -383,18 +409,55 @@ def collect_oneroster_dfs(bucket, key):
     }
 
 
-def compile_models(cli_bucket, cli_key, oneroster_bucket, oneroster_key):
+def collect_quiz_dfs(bucket, key):
+    key_questions = key + "/contents/quiz_questions.csv"
+    key_question_contents = key + "/contents/quiz_question_contents.csv"
+    key_multichoice_answers = key + "/contents/quiz_multichoice_answers.csv"
```
Comment (rnathuji) [inline enclave_mgmt/compile_models.py:415]: Minor-ish (but super important we're consistent :joy: ): We we want `/contents/` or `/content/`? We were discussing the latter [here](https://github.com/openstax/k12/issues/307#issuecomment-1387497365), but maybe this is better. I just want to be sure we explicitly decide on the convention either way :man_shrugging: .

Comment (mjoss17) [reply enclave_mgmt/compile_models.py:415]: Hmm, yea good point. They kind of imply different things - for that reason I think I'd rather go with content. I'll make that tweak to `/content/`

File: enclave_mgmt/compile_models.py [hunk 2]
```diff
@@ -248,6 +268,12 @@ def create_models(output_path, all_raw_dfs):
         courses_df.to_csv(f, index=False)
     with open(f"{output_path}/{MODEL_FILE_ASSESSMENTS}", "w") as f:
         assessments_df.to_csv(f, index=False)
+    with open(f"{output_path}/{MODEL_QUIZ_QUESTIONS}", "w") as f:
+        quiz_questions_df.to_csv(f, index=False)
+    with open(f"{output_path}/{MODEL_QUIZ_QUESTION_CONTENTS}", "w") as f:
+        quiz_question_contents_df.to_csv(f, index=False)
+    with open(f"{output_path}/{MODEL_MULTICHOICE_ANSWERS}", "w") as f:
+        quiz_multichoice_answers_df.to_csv(f, index=True, index_label="id")
```
[the remark this answers is not in the document]
Comment (mjoss17) [reply enclave_mgmt/compile_models.py:276]: I'm happy to do the latter. I also enjoy the aspect of having all enclave-ready files accessible after a single command. The alternative of having the argo workflow grab specific files and stage them while other files get transformed here seems a little more wayward. I'll append the pydantic checks and multichoice index change.

**Review by rnathuji — COMMENTED** [review]
@mjoss17 - This looks good. I just had a couple of super-minor final comments.

**Review by rnathuji — APPROVED** [review]

**Pull request merged**